src/store/selectors/operations.ts

```diff
@@ -49,6 +49,17 @@ export function getRequestVerificationOperation(
   ) as List<RequestVerificationOperation>;
 }
 
+export function operationsForMember(
+  operations: List<Operation>,
+  memberId: MemberId
+) {
+  return operations.filter(
```

Comment:
Let's not solve it in this PR, but created https://github.com/rahafoundation/raha-mobile-app/issues/239 to track our divergent filtering patterns cc @osdiab